OtripleS.Web.Api/Services/Calendars/CalendarService.cs

```diff
@@ -57,6 +57,18 @@ public partial class CalendarService : ICalendarService
 			return storageCalendar;
 		});
 
+		public ValueTask<Calendar> DeleteCalendarByIdAsync(Guid calendarId) =>
```

Comment:
Make sure the Delete function comes at the end after all other methods.
We follow the CRUD operations order, so we start with the Add, Retrieve All, Retrieve, Modify and then Delete, move this function implementation to be below the `Modify` function.